roles/openshift_node/tasks/upgrade.yml

```diff
@@ -41,6 +41,14 @@
   - "/etc/cni/net.d/100-crio-bridge.conf"
   when: crio_update is changed
 
+- name: Ensure crictl.yaml runtime-endpoint is updated
```

Comment:
Ideally, this should live with container_runtime role, but I'm not sure we have a crio upgrade path yet.